cadc-util/src/test/java/org/opencadc/persist/EntityTest.java

```diff
@@ -105,9 +105,28 @@ public void testTemplate() {
     
     @Test
     public void testEntity() {
-        
+        // base: the cadc-inventory-0.x configuration
+        doEntityTest(false, false);
+        doNewVersionTest(false, false);
+    }
+    
+    @Test
+    public void testEntityTruncateDates() {
+        // the caom2-2.4 configuration
+        doEntityTest(true, false);
+        doNewVersionTest(true, false);
+    }
+    
+    @Test
+    public void testEntityDigestFieldNames() {
+        // the cadc-vos-2.x configuration
+        //doEntityTest(false, true);
+        doNewVersionTest(false, true);
+    }
+
```

Comment:
Where's the true, true case?

Reply:
hmmm, we don't do that, but I guess it's the safest and most general. will add.